app/backend/src/couchers/migrations/versions/71efe49ea92f_update_notification_enums_for_event_.py

```diff
@@ -0,0 +1,25 @@
+"""Update notification enums for event cancellation/deletion
+
+Revision ID: 71efe49ea92f
+Revises: 46e6b6a3d9ed
+Create Date: 2024-05-29 00:16:26.791019
+
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "71efe49ea92f"
+down_revision = "46e6b6a3d9ed"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.get_context().autocommit_block():
+        op.execute("ALTER TYPE notificationtopicaction ADD VALUE 'event__cancel'")
+        op.execute("ALTER TYPE notificationtopicaction ADD VALUE 'event__delete'")
+
+
+def downgrade():
+    raise Exception("Can't downgrade")
```

Comment:
Could you please combine the two migrations into one file. And please rebase on develop (you might need to change the down revision to the latest value) to keep a linear alembic history.

Reply:
I combined the migrations now - I merged develop into this branch already and added these migrations on top, so the history is linear.